src/renderer.test.ts

```diff
@@ -217,13 +218,13 @@ describe('Renders different types of elements', () => {
     test('ElementKind.Pullquote', () => {
         const nodes = render(pullquoteElement())
         const pullquote = nodes.flat()[0];
-        expect(getHtml(pullquote)).toContain('<blockquote><p>quote</p></blockquote>');
+        expect(getHtml(pullquote)).toContain('<aside><blockquote><p>quote</p></blockquote></aside>');
     })
 
     test('ElementKind.Pullquote with attribution', () => {
         const nodes = render(pullquoteWithAttributionElement())
         const pullquote = nodes.flat()[0];
-        expect(getHtml(pullquote)).toContain('<blockquote><p>quote</p><cite>attribution</cite></blockquote>');
+        expect(getHtml(pullquote)).toContain('<aside><blockquote><p>quote</p><cite>attribution</cite></blockquote></aside>');
```

Comment:
Have these been pulled in from some other changes?

Reply:
Yes it was pulled in from prev ticket, ill update and check both as the test was written